desktop/lib/messagebundles/default.po

```diff
@@ -4269,6 +4269,25 @@ msgstr ""
 msgid "Enable Distributed Hash Table (DHT)"
 msgstr ""
 
+#: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/options/panes/TorrentConnectionPaneItem.java:42
```

Comment:
@marcelinkaaa nope, we're talking about desktop here. I think there was some StatusLine component already there he found, I've not yet been able to test this branch.